SDWebImage/UIImage+MultiFormat.m

```diff
@@ -7,114 +7,14 @@
  */
 
 #import "UIImage+MultiFormat.h"
-#import "UIImage+GIF.h"
-#import "NSData+ImageContentType.h"
-#import <ImageIO/ImageIO.h>
-
-#ifdef SD_WEBP
-#import "UIImage+WebP.h"
-#endif
+#import "SDWebImageDecoder.h"
 
 @implementation UIImage (MultiFormat)
 
 + (nullable UIImage *)sd_imageWithData:(nullable NSData *)data {
```

Comment:
Maybe not. Our users(including me in my product) still use this one to decoded raw image data. And it's an more easy usage. Keep these public API and call the new `SDWebImageDecoder` in implementation is not a pain.